tests/serial/action-timings/ActionTimingsExplicit.cpp

```diff
@@ -62,8 +62,8 @@ BOOST_AUTO_TEST_CASE(ActionTimingsExplicit)
 
   if (context.isNamed("SolverOne")) {
     BOOST_TEST(action::RecorderAction::records.size() == 2);
-    BOOST_TEST(action::RecorderAction::records.at(0).timing == action::Action::WRITE_MAPPING_PRIOR);
-    BOOST_TEST(action::RecorderAction::records.at(1).timing == action::Action::WRITE_MAPPING_POST);
+    BOOST_TEST(action::RecorderAction::records.at(0).timing == action::Action::READ_MAPPING_PRIOR);
+    BOOST_TEST(action::RecorderAction::records.at(1).timing == action::Action::READ_MAPPING_POST);
```

Comment:
I added `initialize="True"` in this test to actually check whether the branch is entered. The first participant then only reads the initial data. It does now write data at this point.

Reply:
> It does now write data at this point

It does no**t** write data at this point ??

Reply:
Yes. Important difference!